### PR TITLE
Fixed shop.display-item-use-name work properly

### PR DESCRIPTION
**Pull request by jho5245**
now RealDisplayItem.java

### CODE REVIEW

**Comment by Ghost-chu** [comment]
If you add this feature, you need update `DisplayItem` interface and create adapter methods in ArmorStandDisplay.

**Comment by Ghost-chu** [comment]
You must check config value `shop.display-item-use-name` if it disabled, this feature must auto-disable.

And this feature should allow user switch it.

**Comment by jho5245** [comment]
I have tested it, when i disabled the value, the custom name automatically disappears

**Comment by jho5245** [comment]
and making this feature on armor stand is quite difficult way. if custom name of armor stand is visible, the custom name is displayed much above then the location of the shop's chest

**Comment by Ghost-chu** [comment]
I think you not pushed them files to git.
I didn't see you have changed ArmorStandDisplayItem files

**Comment by jho5245** [comment]
I think I can't make this feature on armor stand by myself or this may be very unefficient way to make it work.

**Comment by Ghost-chu** [comment]
yeah it very strange

**Comment by Ghost-chu** [comment]
so maybe you can create a virtual display by ProtocolLib and we can drop fucking armorstand, all owner will love it :D

**Comment by jho5245** [comment]
meh, haven't used ProtcolLib yet, I'll make it someday if I learned how to use that plugin properly. It's too hard for me now :(

**Comment by jho5245** [comment]
and i haven't touched ANY other property of items (like lore ( 20 times same protection-detect) lore)
i just set the customname from itemi18n and set visible to true when config value is true, and visible to false when config value is false. set the config and typed /qs reload, it automatically updated.
I think is done. not need to use other methods to check it is protected

File: src/main/java/org/maxgamer/quickshop/Shop/DisplayItem.java [hunk 1]
```diff
@@ -14,17 +14,17 @@
...
 /**
  * @author Netherfoam A display item, that spawns a block above the chest and
  * cannot be interacted with.
  */
 public interface DisplayItem {
 
+    /**
+    * constant literal to check if item is quickshop display item
+    */
+    static final String QUICKSHOP_DISPLAY_ITEM = "§q§s§ §d§i§s§p§l§a§y";
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/Shop/DisplayItem.java]: Some special build server have bad support for color code. For compatibility i recommend use normal text.

(it not appears on 1.14 but before on 1.13 some legacy a magic build.)

**Pull request merged**